packages/zowe-explorer/src/utils/CertificateWizard.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { Gui, WebView } from "@zowe/zowe-explorer-api";
+import { Disposable, ExtensionContext, OpenDialogOptions, Uri } from "vscode";
+import * as nls from "vscode-nls";
+import { ZoweLogger } from "./LoggerUtils";
+
+// Set up localization
+nls.config({
+    messageFormat: nls.MessageFormat.bundle,
+    bundleFormat: nls.BundleFormat.standalone,
+})();
+const localize: nls.LocalizeFunc = nls.loadMessageBundle();
```

Comment:
When this is ported to vNext, I assume we would want to change this to the `@vscode/l10n` implementation, right? 😋

Reply:
Yep 😋